public/css/style.css

```diff
@@ -2,20 +2,20 @@ h3 {
 	color: dimgrey;
 }
 
-h4 {
-	color: dimgrey;
-}
```

Comment:
Making a change to all `h4` tags is too vague, hmmm... I'll replace the style, but specify a specific class for the `h4` you were styling